polaris-react/src/components/Navigation/Navigation.scss

```diff
@@ -127,6 +136,12 @@ $disabled-fade: 0.6;
     border-bottom-right-radius: var(--p-border-radius-1);
   }
 
+  #{$se23} & {
+    &::before {
+      display: none;
```

Comment:
I think this can be `content: none` so the pseudo element doesn't render. Probably doesn't really matter but less dom elements

Reply:
I can change it. Both should achieve the same result 👍🏻